src/core/grpc_service.proto

```diff
@@ -65,6 +65,12 @@ service GRPCService
   //@@
   rpc Control(ControlRequest) returns (ControlResponse) {}
```

Comment:
This should change to ModelControl, ModelControlResponse, etc. And the endpoint should be /api/modelcontrol. You should do that in a separate change and Guan and review to make sure you get all the cases.

Reply:
Sure

Reply:
PR for the rename from Control to ModelControl is here: #488